src/js/angular/graphexplore/controllers/dependencies-chord.controller.js

```diff
@@ -51,18 +51,27 @@ DependenciesChordCtrl.$inject = ['$scope', '$rootScope', '$repositories', 'toast
 function DependenciesChordCtrl($scope, $rootScope, $repositories, toastr, $timeout, GraphDataRestService, UiScrollService, ModalService, LocalStorageAdapter, RDF4JRepositoriesRestService, $translate) {
 
     let timer = null;
+    const MAX_LOADED_GRAPHS = 1000;
```

Comment:
Put documentation what is this for. It's not quite clear from the const name

Reply:
Done